core/src/main/java/com/opensymphony/xwork2/interceptor/CSRFPreventionInterceptor.java

```diff
@@ -0,0 +1,44 @@
+package com.opensymphony.xwork2.interceptor;
+
+import com.opensymphony.xwork2.ActionContext;
+import com.opensymphony.xwork2.ActionInvocation;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CSRFPreventionInterceptor extends AbstractInterceptor {
+
+    private List<String> exemptedPaths = new ArrayList<String>();
+    private DefaultResourceIsolationPolicy fmPolicy = new DefaultResourceIsolationPolicy();
```

Comment:
This reference should be of type ResourceIsolationPolicy so we can swap it in the future for other implementations of the interface
```suggestion
    private List<String> exemptedPaths = new ArrayList<String>();
    private ResourceIsolationPolicy fmPolicy = new DefaultResourceIsolationPolicy();
```

Reply:
let's also call it resourceIsolationPolicy, please! Remember: Fetch Metadata is made up of a collection of Resource Isolation policies (in theory one or more), but we're going with just the one for now